graphics/partio/Portfile

```diff
@@ -30,3 +32,24 @@ depends_build-append \
                     port:doxygen \
                     port:swig-python
 depends_lib         port:zlib
+
+post-destroot {
+    if {![file exists ${destroot}${prefix}/share/${name}/py]} {
+        xinstall -d ${destroot}${prefix}/share/${name}/py
+    }
+    copy ${worksrcpath}/src/py/example \
+         ${destroot}${prefix}/share/${name}/py
+}
+
+variant tests description {Build tests} {
+    depends_lib-append      port:gtest
```

Comment:
I'll need to do some checking to see where exactly osl actually uses gtest. I might be able to restrict the dependency to a specific phase as you suggest.

Reply:
So, unfortunately, I can't restrict the dependency to `depends_test-append`. Because the unit tests get generated as executable binaries in the final output products, which the user can run themselves, the tests won't build successfully unless the dependency is specified as `depends_lib-append`.